packages/opentelemetry-metrics/src/Metric.ts

```diff
@@ -193,3 +198,38 @@ export class GaugeMetric extends Metric<GaugeHandle>
     this.getHandle(labelSet).set(value);
   }
 }
+
+export class MeasureMetric extends Metric<MeasureHandle>
+  implements Pick<types.MetricUtils, 'record'> {
+  protected readonly _absolute: boolean;
+
+  constructor(
+    name: string,
+    options: MetricOptions,
+    private readonly _onUpdate: Function
+  ) {
+    super(
+      name,
+      options,
+      options.valueType === types.ValueType.DOUBLE
+        ? MetricDescriptorType.GAUGE_DOUBLE
```

Comment:
I am not sure whether this is correct `MetricDescriptorType`  for the measure. I will spend some time on specs to read more about the measure.

Reply:
I believe it would be `MEASURE_DOUBLE` and `MEASURE_INT64`